src/main/java/io/javalin/serversentevent/EventSource.kt

```diff
@@ -0,0 +1,23 @@
+package io.javalin.serversentevent
+
+import io.javalin.Context
+
+class EventSource(private val emitter: Emitter, private val context: Context) {
+
+    private var close: SseClose? = null
+
+    fun onClose(close: SseClose) {
```

Comment:
What happened to `onConnect` ?

Reply:
It's redundant, because if you enter it's because it's connected.